tests/frontend/test_engine.py

```diff
@@ -46,7 +46,7 @@ def starship_engine(monkeypatch):
     """
```

Comment:
I have a feeling this file might have conflicts when we merge #294

Reply:
Potentially 🤔

Reply:
I believe so - it seems that the `StarshipEngine` tests are mocking `APIClient`, which is probably going to lead to a conflict as `APIClient` is removed in https://github.com/XanaduAI/strawberryfields/pull/294.